test/cql-pytest/test_tablets.py

```diff
@@ -91,11 +91,11 @@ def test_tablets_can_be_explicitly_disabled(cql, skip_without_tablets):
         res = cql.execute(f"SELECT * FROM system_schema.scylla_keyspaces WHERE keyspace_name = '{keyspace}'")
         assert len(list(res)) == 0, "tablets replication strategy turned on"
 
-
-def test_alter_changes_initial_tablets(cql, skip_without_tablets):
+@pytest.mark.xfail(reason="issue #18801: Chaning initial tablets with ALTER keyspace needs `WITH replication = {`")
+def test_alter_without_replication_options_changes_initial_tablets(cql, skip_without_tablets):
     ksdef = "WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': 1} AND tablets = {'initial': 1};"
     with new_test_keyspace(cql, ksdef) as keyspace:
-        cql.execute(f"ALTER KEYSPACE {keyspace} WITH replication = {{'class': 'NetworkTopologyStrategy', 'replication_factor': 1}} AND tablets = {{'initial': 2}};")
+        cql.execute(f"ALTER KEYSPACE {keyspace} AND tablets = {{'initial': 2}};")
```

Comment:
I don't understand what I'm seeing here... You took a passing test which, if I understand correctly, should continue to pass, and "broke" it to fail because of #18801?
I think what you should have done is to duplicate this test - have the old one which passes, and a new one, which fails on #18801.